releasetasks/tcutils.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+from taskcluster.utils import slugId
+
+
+def stable_slug_id():
+    """Returns a closure which can be used to generate stable slugIds.
+    Stable slugIds can be used in a graph to specify task IDs in multiple
+    places without regenerating them, e.g. taskId, requires, etc.
+    """
+    _cache = {}
+
+    def closure(name):
+        if name not in _cache:
+            _cache[name] = slugId()
+        return _cache[name]
+
+    return closure
```

Comment:
This can be replaced whenever https://github.com/taskcluster/taskcluster-client.py/pull/20 is released